app/web/markdown/blog.md

```diff
@@ -6,6 +6,14 @@ title: Blog
 
 If you'd like to contribute to the blog, please [sign up](/volunteer) and let us know!
 
+## [A new chapter: Couchers is officially out of Beta!](/blog/2025/07/01/releasing-couchers-v1)
+
+2025/07/01 by Aapeli.
+
+We are out of Beta and in v1: we're releasing a new strategy around safe & active community instead of bashing our competitors, a fancy redesigned landing page, and a bunch of new features to make core couch surfing functionality better!
+
```

Comment:
```suggestion
We are out of Beta and into version 1: learn what this means, our new strategy around safe & active community instead of bashing our competitors, a fancy redesigned landing page, and a bunch of new features to make core couch surfing functionality better!

```